src/Microsoft.DocAsCode.Build.Engine/TemplatePreprocessors/TemplateUtility.cs

```diff
@@ -54,5 +58,33 @@ public string GetHrefFromRoot(string originalHref, string sourceFileKey)
             }
             return file.UrlEncode().ToString() + UriUtility.GetQueryStringAndFragment(originalHref);
         }
+
+        public string Markup(string markdown, string sourceFileKey)
```

Comment:
As it's hard to pass it here, so preserve as a parameter

---
In reply to: [113619942](https://github.com/dotnet/docfx/pull/1606#discussion_r113619942) [](ancestors = 113619942)